DATA/MISSIONS/npcships.ini

```diff
@@ -866,7024 +1241,9022 @@ npc_class = lawful, freighter
+level = d3
+loadout = gd_z_fighter4_loadout_d3_b
+ship_archetype = cv_ship_fighter_light01_npc
 state_graph = FIGHTER
-pilot = pilot_pirate_ace
-npc_class = unlawful, class_medium, class_heavy, class_fighter, class_mission, d19
-
-;; Wolfshund
+pilot = pilot_pirate_med
+npc_class = lawful, class_light, class_fighter, class_mission, d3
 
 [NPCShipArch]
-nickname = fc_m_pi_elite_d13-15
-level = d13
-loadout = fc_m_pi_elite_loadout_d13-15
-ship_archetype = pi_ship_fighter_heavy01_npc
+nickname = gd_z_fighter4_d4_a
+level = d4
+loadout = gd_z_fighter4_loadout_d4_a
+ship_archetype = cv_ship_fighter_light01_npc
 state_graph = FIGHTER
 pilot = pilot_pirate_hard
-npc_class = unlawful, class_medium, class_heavy, class_fighter, class_mission, d13, d14, d15
+npc_class = lawful, class_light, class_fighter, class_mission, d4
 
 [NPCShipArch]
-nickname = fc_m_pi_elite_d16-18
-level = d16
-loadout = fc_m_pi_elite_loadout_d16-18
-ship_archetype = pi_ship_fighter_heavy01_npc
+nickname = gd_z_fighter4_d4_b
+level = d4
+loadout = gd_z_fighter4_loadout_d4_b
+ship_archetype = cv_ship_fighter_light01_npc
 state_graph = FIGHTER
-pilot = pilot_pirate_ace
-npc_class = unlawful, class_medium, class_heavy, class_fighter, class_mission, d16, d17, d18
+pilot = pilot_pirate_hard
+npc_class = lawful, class_light, class_fighter, class_mission, d4
 
 [NPCShipArch]
-nickname = fc_m_pi_elite_d19
-level = d19
-loadout = fc_m_pi_elite_loadout_d19
-ship_archetype = pi_ship_fighter_heavy01_npc
+nickname = gd_z_fighter4_d5_a
+level = d5
+loadout = gd_z_fighter4_loadout_d5_a
+ship_archetype = cv_ship_fighter_light01_npc
 state_graph = FIGHTER
 pilot = pilot_pirate_ace
-npc_class = unlawful, class_medium, class_heavy, class_fighter, class_mission, d19
+npc_class = lawful, class_light, class_fighter, class_mission, d5
 
 [NPCShipArch]
-nickname = fc_m_pi_elite_d16-18smuggler
-level = d16
-loadout = fc_m_pi_elite_loadout_d16-18
-ship_archetype = pi_ship_fighter_heavy01_npc
+nickname = gd_z_fighter4_d5_b
+level = d5
+loadout = gd_z_fighter4_loadout_d5_b
+ship_archetype = cv_ship_fighter_light01_npc
 state_graph = FIGHTER
 pilot = pilot_pirate_ace
-npc_class = unlawful, class_fighter_smuggler, d1, d2, d3, d3, d4, d5, d6, d7, d8, d9, d10, d11, d12, d13, d14, d15, d16, d17, d18, d19
-
-;;Mulli 
+npc_class = lawful, class_light, class_fighter, class_mission, d5
 
-[NPCShipArch]
-nickname = fc_m_pi_freighter_d1-8
-loadout = fc_m_pi_freighter_loadout01
-level = d8
-ship_archetype = pi_ship_freighter01_npc
-pilot = freighter_easy
-state_graph = FIGHTER
-npc_class = unlawful, class_freighter, class_freighter_smuggler, d1, d2, d3, d3, d4, d5, d6, d7, d8
+;; Falcon
 
 [NPCShipArch]
-nickname = fc_m_pi_freighter_d9-19
-loadout = fc_m_pi_freighter_loadout02
-level = d20
-ship_archetype = pi_ship_freighter01_npc
-pilot = freighter_ace
+nickname = gd_z_fighter5_d2_a
+level = d2
+loadout = gd_z_fighter5_loadout_d2_a
+ship_archetype = cv_ship_fighter_medium01_npc
 state_graph = 
```

Comment:
Are Bosses used?

Reply:
currently not used (placeholder) but i will use them in future NPC Adjustment phases